src/commands/start-client.js

```diff
@@ -17,7 +17,8 @@ const APP_CONFIG_PATH = path.join(APP_ROOT, "src", "config", "application.js");
 const ASSET_PATH = getAssetPath();
 
 const IS_PRODUCTION = process.env.NODE_ENV === "production";
-const PORT = 8888;
+
+const appConfig = require(APP_CONFIG_PATH).default;
```

Comment:
This should fall back to `8888` if it isn't defined in the app config. That way old apps wont be broken